client/src/routes/app/[country]/[map]/[year]/initialize/LoadRegions.ts

```diff
@@ -17,7 +17,7 @@ function fillRegion(regionID: string, increment?: boolean) {
 			count: region.value,
 			margin: 0
 		};
-		if (currentCandidate.candidate.id === selectedCandidate.id && increment === true) {
+		if (currentCandidate.candidate.id === selectedCandidate.id && increment === undefined) {
```

Comment:
This is a bit confusing, because now if you pass in true, it will not increment.

I would recommend to just make the parameter required. And then saying ```increment === true```

Reply:
Thought about this earlier, held off. Will change.